float32 purity_level 
float32 clean_water_available 
string status   # "OK" or "not OK"